app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -59,7 +59,7 @@ examples:
     description: |
       An `id` is optional as it's automatically generated, but it can be supplied if a specific `id` is required.
 
-      The `id` must be **unique** across the domain of your service - this is because as the open / closed state of individual instances of the accordion persists across page loads using `localStorage`.
+      The `id` must be unique across the domain of your service - this is because the open / closed state of individual instances of the accordion persists across page loads using `localStorage`.
 
       Used as an `id` in the HTML for the accordion as a whole, and also as a prefix for the `id`s of the section contents and the buttons that open them, so that those `id`s can be the target of `aria-labelledby` and `aria-control` attributes.
     data:
```

Comment:
> _This sentence is over 25 words, which is our recommended maximum length.
What’s the (grammatical) subject of this sentence?_

Reply:
That's shorter, which is cool, but I wasn't sure whether it was the `id` or the service that uses `localStorage`...

Maybe we could break it up into 2 sentences. Something like:

> The `id` must be unique across the domain of your service. This is because the open or closed state of individual instances of the accordion uses `localStorage` to persist across page loads.

Reply:
Sure, updated to this

![image](https://user-images.githubusercontent.com/71266765/142912863-903d5e65-e894-49fb-b0ff-899c7d9e7805.png)